Formula/phoronix-test-suite.rb

```diff
@@ -31,7 +33,22 @@ def post_install
   end
 
   test do
-    cd pkgshare
-    assert_match version.to_s, shell_output("#{bin}/phoronix-test-suite version")
+    on_macos { cd pkgshare }
+
+    # Work around issue directly running command on Linux CI by using spawn.
+    # Error is "Forked child process failed: pid ##### SIGKILL"
+    require "pty"
+    output = ""
+    PTY.spawn(bin/"phoronix-test-suite", "version") do |r, _w, pid|
```

Comment:
Not sure why the error in comment happens. Tried to re-run CI multiple times and always hits it. Maybe some odd situation of children processes not properly terminating.

I tried locally on copy of Linux Docker image and didn't hit this.

Since I couldn't find a way to deal with it, I just went with spawning a separate process to handle lifespan/exceptions separately.